installer/x86_64/install.sh

```diff
@@ -536,7 +536,7 @@ trap_push "rm $grub_cfg || true"
 [ -r ./platform.conf ] && . ./platform.conf
 
 DEFAULT_GRUB_SERIAL_COMMAND="serial --port=${CONSOLE_PORT} --speed=${CONSOLE_SPEED} --word=8 --parity=no --stop=1"
-DEFAULT_GRUB_CMDLINE_LINUX="console=tty0 console=ttyS${CONSOLE_DEV},${CONSOLE_SPEED}n8 quiet"
+DEFAULT_GRUB_CMDLINE_LINUX="console=tty0 console=ttyS${CONSOLE_DEV},${CONSOLE_SPEED}n8 quiet crashkernel=256M"
```

Comment:
This will reserve 256M of RAM by default. The idea is to set crashkernel=xxx value only if kdump is enabled. Can you please elaborate why we need this?

Reply:
@shlomibitton how about `crashkernel=auto`?

Reply:
The below value is observed to be working on a lot of platforms that I tested. 
crashkernel="0M-2G:256M,2G-4G:320M,4G-8G:384M,8G-:448M"

This https://github.com/Azure/sonic-buildimage/pull/6122 implements logic to set kdump configuration according to the default configuration. Ideally, we will not require setting crashkernel value explicitly here.

Reply:
@rajendra-dendukuri you are correct, this is not required since it is handled in the PR you referenced, I removed it.